superset-frontend/packages/superset-ui-core/src/chart-composition/ChartFrame.tsx

```diff
@@ -28,7 +28,7 @@ type Props = {
   contentWidth?: number;
   contentHeight?: number;
   height: number;
-  renderContent: ({
+  renderContent?: ({
```

Comment:
This should be mandatory. There is no reason for this component to exist if this function is not given